web/src/locales/zh/data.json

```diff
@@ -944,6 +944,9 @@
     "Select a photo...": "选择图片...",
     "Set Password": "设置密码",
     "Set new profile picture": "设置新头像",
+    "Set new IdCardFront picture": "上传身份证正面照片",
```

Comment:
Auto update i18n for frontend and backend: https://casdoor.org/docs/contributing#how-to-update-i18n-data